grafanalib/core.py

```diff
@@ -1054,6 +1072,37 @@ def to_json_data(self):
         }
 
 
+@attr.s
+class AlertList(object):
+    """Generates the AlertList Panel."""
+
+    description = attr.ib(default="")
+    id = attr.ib(default=None)
+    limit = attr.ib(default=DEFAULT_LIMIT)
+    links = attr.ib(default=attr.Factory(list))
+    onlyAlertsOnDashboard = attr.ib(default=True, validator=instance_of(bool))
```

Comment:
Please use `boolean(True)` rather than the `attr.ib` expression.

Reply:
My bad. I was looking at a local branch of mine, assuming it was current master. Please disregard this & similar comments.